packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -45,6 +45,8 @@ import { localVideoTileWithControlsContainerStyle, LOCAL_VIDEO_TILE_ZINDEX } fro
 import { _ICoordinates, _ModalClone } from './ModalClone/ModalClone';
 import { _formatString } from '@internal/acs-ui-common';
 import { _LocalVideoTile } from './LocalVideoTile';
+// eslint-disable-next-line no-restricted-imports
+import { VideoRenderingControlOptions } from '../../../react-composites/src/composites/CallComposite/types/VideoRenderingControlOptions';
```

Comment:
`react-composites` dir has a dependency on `react-components` but react-components cannot use code from the composites dir.
Instead move any interfaces defined into the `react-components` dir and reference those when used in the react-composites dir. But `VideoRenderingControlOptions` should be able to be simply removed from the composites dir per https://github.com/Azure/communication-ui-library/pull/1954/files#r889292086

Reply:
We have decided to just remove videoRenderingControlOptions